blogs/posts/2024-01-16_nearest_neighbour.qmd

```diff
@@ -0,0 +1,181 @@
+---
+title: "Nearest neighbour imputation"
+author: "Jacqueline Grout with Tom Jemmett"
```

Comment:
I think we probably want multiple authors separated in the YAML like:

``` r
authors:
  - Jacqueline Grout
  - Tom Jemmett
```

This will make it easier to filter blog posts by author. But I think this was _written_ by you (correct me if wrong), so maybe you only need your name on it as the author of this particular blog? No reason you can't shout-out Tom in the post itself.